cypress/e2e/components/filings/correctionFilings.cy.ts

```diff
@@ -1,6 +1,6 @@
 import { allFilings } from '../../../fixtures/filings/allFilings'
 import { administrativeDissolution } from '../../../fixtures/filings/dissolution/administrativeDissolution'
-import { devBCReg } from '../../../fixtures/origins'
```

Comment:
This generated an error in VS Code. Hopefully allowing the import (see tsconfig file) will now allow line 32 (the previous test error) to pass.